sections/real-life-examples/test-front-end-with-integration-back-end-with-e2e.md

```diff
@@ -131,7 +130,7 @@ if (isLocalHost()) {
 
 ```
 
-For Cypress users, recently [cypress-skip-test plugin](https://github.com/cypress-io/cypress-skip-test) added support for environment variables. Per CTO Gleb Bahmutov, this may soon become a standard for specifying environmental constraints on a per spec or per test basis. 
+For Cypress users, recently [cypress-skip-test plugin](https://github.com/cypress-io/cypress-skip-test) added support for environment variables. Per CTO Gleb Bahmutov, this may soon become a standard for specifying environmental constraints on a per spec or per test basis.
```

Comment:
It did become standard. You can control all of it with an object passed as the last argument to a describe/context or it block.

Reply:
you can delete this comment... :)

Reply:
Done!